cmd/podman/libpodruntime/runtime.go

```diff
@@ -136,7 +136,9 @@ func GetRuntimeWithStorageOpts(c *cli.Context, storageOpts *storage.StoreOptions
 	if c.GlobalIsSet("default-mounts-file") {
 		options = append(options, libpod.WithDefaultMountsFile(c.GlobalString("default-mounts-file")))
 	}
-	options = append(options, libpod.WithHooksDir(c.GlobalString("hooks-dir-path"), c.GlobalIsSet("hooks-dir-path")))
+	if c.GlobalIsSet("hooks-dir-path") {
+		options = append(options, libpod.WithHooksDir(c.GlobalString("hooks-dir-path")))
```

Comment:
`--hooks-dir-path` was previously an undocumented option used only for internal testing (e.g. see [here][1]).  Are we changing that?  If so, I think we want to add documentation at the same time.

[1]: https://github.com/containers/libpod/pull/155#discussion_r162978659